Dockerfile

```diff
@@ -2,10 +2,12 @@ FROM ruby:2.5
 
 WORKDIR /app
 
-ADD Gemfile Gemfile.lock /app/
-RUN bundle update
+COPY Gemfile Gemfile.lock /app/
+ENV BUNDLER_VERSION 2.0.1
+RUN gem install bundler
+RUN bundle install --jobs 20 --retry 5
```

Comment:
I took this workaround based on the feedback [here](https://stephencodes.com/upgrading-ruby-dockerfiles-to-use-bundler-2-0-1/)
I guess it will always run from scratch as the CI doesn't really cache builds between PRs or other branches. So no strong opinion about this

Reply:
If it works then it works 😄